### PR TITLE
Add Biswashree to list of maintainers

### PR DESCRIPTION
**Pull request by biscuitdey**
# Description

Add me (Biswashree Dey) as a code maintainer to the repo

- I have contributed to BRI-1 by creating the Excel Plugin
- I am part of the BRI-3 team, working specifically on the zero knowledge component.
- I am part of the Interop WG, helping the team prepare for upcoming execution.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Request to be added as a Code Owner/Maintainer

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] I commit to abide by the Responsibilities of Code Owners/Maintainers.

### CODE REVIEW

**Review by ognjenkurtic — DISMISSED** [review]

**Review by Kasshern — DISMISSED** [review]

**Review by Manik-Jain — DISMISSED** [review]
Please resolve the conflict

**Review by Manik-Jain — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 # These owners will be the default owners for everything in
+
 # the repo. Unless a later match takes precedence,
+
 # the following owers will be requested for
+
 # review when someone opens a pull request.
-*       @skarred14 @ognjenkurtic @therecanbeonlyone1969 @Manik-Jain @skosito @Kasshern @ybittan
+
+-       @skarred14 @ognjenkurtic @therecanbeonlyone1969 @fosgate29 @Manik-Jain @skosito @Kasshern @ybittan @biscuitdey
```
Comment (ognjenkurtic) [inline .github/CODEOWNERS:15]: Please remove @fosgate29 as he was just removed form the list yesterday or today

File: MAINTAINERS.md [hunk 2]
```diff
@@ -25,6 +25,7 @@ Code and Systems maintainers
 - [Lucas Rodriguez Benitez](https://github.com/LucasRodriguez)
 - [Anais Ofranc](https://github.com/Consianimis)
 - [Hamza Tokuchi](https://github.com/Meuko) - [BlockLab](https://www.blocklab.nl/)
+- [Biswashree Dey](https://github.com/biscuitdey)
```
Comment (skosito) [inline MAINTAINERS.md:28]: this list is not updated, lets not add to it, probably should be removed completely?

File: .github/CODEOWNERS (hunk 1 above)
Comment (biscuitdey) [reply .github/CODEOWNERS:15]: Done.

File: MAINTAINERS.md (hunk 2 above)
Comment (biscuitdey) [reply MAINTAINERS.md:28]: Done.

**Review by ognjenkurtic — APPROVED** [review]

**Review by Kasshern — APPROVED** [review]

**Review by Ybittan — APPROVED** [review]
lgtm

**Pull request merged**